content/community.html

```diff
@@ -389,6 +389,21 @@ <h2 id="contributing">Contributing</h2>
   <li>Contributing code to the project (<a href="https://issues.apache.org/jira/issues/?jql=project+%3D+HUDI+AND+component+%3D+newbie">newbie JIRAs</a>)</li>
 </ul>
 
+<h2 id="become-a-committer">Become a Committer</h2>
+
+<p>We are always looking for strong contributors, who can become <a href="https://www.apache.org/dev/committers.html">committers</a> on the project. 
+Committers are chosen by a majority vote of the Apache Hudi <a href="https://www.apache.org/foundation/how-it-works.html#pmc-members">PMC</a>, after a discussion on their candidacy based on the following criteria (not exclusive/comphrehensive).</p>
+
+<ul>
+  <li>Embodies the ASF model code of <a href="https://www.apache.org/foundation/policies/conduct.html">conduct</a></li>
+  <li>Has made significant technical contributions such as submitting PRs, filing bugs, testing, benchmarking, authoring HIPs, providing feedback/code reviews (+ more).</li>
+  <li>Has helped the community over a few months, by answering questions on support channels above/triaging issues/jiras.</li>
```

Comment:
Has helped the community over a few months, by answering questions on support channels and triaging issues/jiras ?